tests/test_filtering.py

```diff
@@ -300,6 +302,102 @@ class Meta:
         self.assertQuerysetEqual(f.qs, [2], lambda o: o.pk)
 
 
+@unittest.skipIf(
+    django.VERSION < (1, 8), "version does not support DurationField")
```

Comment:
btw, this check isn't really necessary since django 1.8 is the minimum supported version